lexnlp/ml/tests/test_model_io.py

```
@@ -0,0 +1,324 @@
+
+    def test_loads_cloudpickle_file(self, tmp_path: Path) -> None:
+        import cloudpickle
+
+        path = tmp_path / "model.cloudpickle"
+        with path.open("wb") as fh:
+            cloudpickle.dump({"cp": 1}, fh)
+        assert _load_legacy(path) == {"cp": 1}
+
+
+# ---------------------------------------------------------------------------
+# load_model – routing logic
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)